sympy/printing/str.py

```diff
@@ -713,6 +713,13 @@ def _print_Complement(self, expr):
     def _print_Quantity(self, expr):
         return "%s" % expr.name
 
+    def _print_Quaternion(self, expr):
+        i, j, k = symbols('i j k')
+        # TODO: This expression is potentially confusing,
+        # shall we print it as `Quaternion( ... )`?
```

Comment:
I think that is what `ReprPrinter` should do.

Reply:
I think u mixed up the latex and str code.

Reply:
Oh, yes. Sorry about that. I just wanted to show where to put the code.